apps/condo/lang/en/en.json

```
@@ -611,6 +613,17 @@
   "pages.condo.meter.EndDate": "End date",
   "pages.condo.meter.EnterMeterNumber": "Enter number",
   "pages.condo.meter.EnterPlace": "Enter place",
+  "pages.condo.settings.hint.propertyHint": "Property hint",
+  "pages.condo.settings.hint.addAllProperties": "Add all properties",
+  "pages.condo.settings.hint.createTicketHint": "Create ticket hint",
+  "pages.condo.settings.hint.newTicketHint": "New hint",
+  "pages.condo.settings.hint.editTicketHint": "Edit hint",
+  "pages.condo.settings.hint.alert.title": "For some properties of this organization there are already hints",
```

If it is a validation, maybe to rename keys accordingly?

```suggestion
  "pages.condo.settings.hint.validation.alreadyPresented": "For some properties of this organization there are already hints",
```

---

No, it's not validation, it's helper alert